lib/resources/file.rb

```diff
@@ -44,7 +44,7 @@ def initialize(path)
     %w{
       type exist? file? block_device? character_device? socket? directory?
       symlink? pipe? mode mode? owner owned_by? group grouped_into?
-      link_path linked_to? mtime size selinux_label immutable?
+      link_path linked_to? mtime size size_lines selinux_label immutable?
```

Comment:
Thank you. I didn't know you could do that with content or other matchers inside its.  Even afterwards searching using google I can't find any similar examples.  

It's a minor edge case but is there anyway to put code like this "content ? content.lines.count : 0"  inside the its().  I'd like to return size 0 in the case where a file does not exist.